build.sbt

```diff
@@ -627,6 +627,7 @@ lazy val integrationTests = (project in file("integration-tests"))
     javaOptions ++= Seq(
       "--add-exports=java.base/sun.nio.ch=ALL-UNNAMED",
     ),
+    javafmtCheckSettings,
```

Comment:
why not add integrationTests to the root as well?
wouldnt that solve a lot of the problems?